README.md

```diff
@@ -22,17 +20,112 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Run the install generator to get the config initializer, the error template, and the configured middleware per environment
+
+```
+$ rails g open_stax:rescue_from:install
+```
+
+Declare that you want to use the openstax exception rescuer in your controller, preferably your `ApplicationController`
+
+```ruby
+class ApplicationController < ActionController::Base
+
+  # ...
+
+  use_openstax_exception_rescue
+
+  # ...
+
+  # Override the before and after hooks if you want to
+  # (See 'Controller before/after hooks')
+  #
+  # private
+  # def before_openstax_exception_rescue(wrapped_exception)
+  #   # noop
+  # end
+  #
+  # def after_openstax_exception_rescue(wrapped_exception)
+  #   respond_to do |f|
+  #     f.xml { render text: "I respond strangely to the XML format!",
+  #                    status: wrapped_exception.status }
+  #   end
+  # end
+end
+```
+
+## Configuration
+
+This configuration, which is placed in `./config/initializers/openstax_rescue_from.rb` by the install generator, shows the defaults:
+
+```ruby
+OpenStax::RescueFrom.configure do |config|
+  config.raise_exceptions = false          # Rails users may wish to use Rails.application.config.consider_all_requests_local to decide this
+  config.system_logger = Rails.logger      # Any logger that takes a string in an #error method will work
+  config.notifier = ExceptionNotifier      # Any notifier that takes an exception in a #notify_exception method will work
+  config.html_template_path = 'errors/any' # The template path for the HTML response
+  config.layout_name = 'application'       # The layout name for the HTML response
+
+  # Of course, you can append to the following lists and maps:
+
+  # config.exception_status_codes['ArgumentError'] = :unprocessable_entity
+
+  # config.friendly_status_messages[:bad_request] = 'Your request was not good.'
```

Comment:
(like we discussed), would be nice to bundle all the messages, codes, extras into one call for each kind of exception configured into the gem.